app/scripts/modules/core/pipeline/config/stages/jenkins/jenkinsStage.js

```diff
@@ -102,6 +104,31 @@ module.exports = angular.module('spinnaker.core.pipeline.stage.jenkinsStage', [
       }
     }
 
+    this.failureOptionChanged = function() {
+      if ($scope.viewState.failureOption === 'fail') {
+        $scope.stage.failPipeline = true;
+        $scope.stage.continuePipeline = false;
+      } else if ($scope.viewState.failureOption === 'stop') {
+        $scope.stage.failPipeline = false;
+        $scope.stage.continuePipeline = false;
+      } else if ($scope.viewState.failureOption === 'ignore') {
+        $scope.stage.failPipeline = false;
+        $scope.stage.continuePipeline = true;
+      }
+    };
+
+    this.failureOptionInit = function() {
+      var initValue = '';
+      if ($scope.stage.failPipeline === true && $scope.stage.continuePipeline === false) {
+        initValue = 'fail';
+      } else if ($scope.stage.failPipeline === false && $scope.stage.continuePipeline === false) {
+        initValue = 'stop';
+      } else if ($scope.stage.failPipeline === false && $scope.stage.continuePipeline === true) {
+        initValue = 'ignore';
+      }
+      return initValue;
+    };
```

Comment:
Since this method is only called once via the `ng-init` code in the template, it doesn't need to be exposed on the controller and can be rewritten as:
```javascript
function initializeFailureOption() {
  let initValue = '';
  if ($scope.stage.failPipeline === true && $scope.stage.continuePipeline === false) {
    initValue = 'fail';
  } else if ($scope.stage.failPipeline === false && $scope.stage.continuePipeline === false) {
    initValue = 'stop';
  } else if ($scope.stage.failPipeline === false && $scope.stage.continuePipeline === true) {
    initValue = 'ignore';
  }
  $scope.viewState.failureOption = initValue;
}

initializeFailureOption();
```
This keeps the controller a little cleaner, and lets you get rid of the `ng-init` attribute in the template.